### ISSUE TITLE
static return type check still not working as expected? 

### ISSUE THREAD

**Issue by bogdandubyk** [squizlabs/PHP_CodeSniffer#3329]
Related to [this](https://github.com/squizlabs/PHP_CodeSniffer/issues/3277). I'm still getting error like 
```
 40 | ERROR | [x] Scope keyword "static" must be followed by a single space; found newline
    |       |     (Squiz.WhiteSpace.ScopeKeywordSpacing.Incorrect)

```
for the static return type, btw I tried to remove `<rule ref="Squiz.WhiteSpace.ScopeKeywordSpacing"/>` rule from my `phpcs.xml` and still see that error 

OS: MacOS 10.15
PHP: 8
PHPCS: 3.6.0

**Comment by jrfnl** [comment]
@bogdandubyk It was fixed in PHPCS 3.6.0 and I just confirmed that the code sample you refer to does not give an error for the PHPCS native sniff when on PHPCS 3.6.0 (or `master`).

Could you run PHPCS with `-s` ? That should show you which sniff the error message is coming from. Might be an external standard which hasn't been updated yet.

Again: Alternatively there may be multiple PHPCS installs on your system and the one being used to scan could be a different one from the Composer installed version.
Try running both below commands to see if the output is different:
```bash
phpcs --version
vendor/bin/phpcs --version
```

**Comment by bogdandubyk** [comment]
umm, okay something weird going on with my local/docker setup, I have version 3.6.0 in my composer also I'm pretty sure on Friday 3.6 was installed as I checked it, but today when I run my docker env it's again 3.5.8. So after I erase the vendors folder and re-run `composer install`, the composer install version 3.6 and now all is working. Sorry for bothering!

**Comment by gsherwood** [comment]
Mystery solved :) Thanks for posting the solution.
